processmanager/manager.go

```diff
@@ -268,17 +268,16 @@ func (pm *ProcessManager) maybeNotifyAPMAgent(
 	rawTrace *host.Trace, umTraceHash libpf.TraceHash, count uint16,
 ) string {
 	pm.mu.RLock()
+	// Keeping the lock until end of the function is needed because inner map can be modified concurrently (by synchronizeMappings/newFrameMapping).
```

Comment:
nit - align with the rest of the code style:

```suggestion
	// Keeping the lock until end of the function is needed because inner map can be modified
	// concurrently (by synchronizeMappings/newFrameMapping).
```